src/components/DatePicker/index.native.js

```diff
@@ -103,37 +113,19 @@ class DatePicker extends React.Component {
                 />
                 <Popover
                     isVisible={this.state.isPickerVisible}
-                    onClose={this.selectDate}
+                    onClose={this.hidePicker}
+                    fromSidebarMediumScreen
+                    anchorPosition={{
+                        // calculating absolute position of popover with additional space of 10px from the input
+                        top: this.state.pickerLayout.height + this.state.spaceFromTop + StatusBar.currentHeight + 10,
+                        left: 20,
+                    }}
                 >
-                    <View style={[
-                        styles.flexRow,
-                        styles.justifyContentBetween,
-                        styles.borderBottom,
-                        styles.pb1,
-                        styles.ph4,
-                    ]}
-                    >
-                        <Button
-                            title={this.props.translate('common.reset')}
-                            color={themeColors.textError}
-                            onPress={this.reset}
-                        />
-                        <Button
-                            title={this.props.translate('common.done')}
-                            color={themeColors.link}
-                            onPress={this.selectDate}
-                        />
+                    <View style={{width: this.state.pickerLayout.width}}>
```

Comment:
same here - inline style

Reply:
I think there will be no considerable difference if you will rewrite that to a variable, as width changes often so that the object will still be updated